specification/crds/go/pkg/apis/serverlessworkflow/v1alpha2/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,162 @@
+// +build !ignore_autogenerated
```

Comment:
Do we want to add the generated code in the repo too or is some automated build process possible? I'm thinking again like in node where we don't add the generated js typically in the source repo.

Reply:
We need this generated code to be able to implement `reflect.DeepEquals` in the object types we are defining here. It's useful for our clients.

Reply:
ok :)